### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cheesinglee**
This is a leiningen plugin which provides two tasks:

`lein test-repeat n test-ns1 test-ns2....`
Repeats the given tests n times to expose any non-deterministic effects

`lein test-children prefix [interactive?]`
Runs all unit tests in namespaces with the given prefix, can be run interactively to select from the list of matching namespaces.

### CODE REVIEW

File: src/leiningen/test_repeat.clj [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+(ns leiningen.test-repeat
+  "Repeat a test multiple times"
+  (:require [leiningen.core.main :as lein]
+            [leiningen.core.eval :as lein-eval]
+            [leiningen.core.project :as lein-project]))
+
+(defn test-repeat
+  "Repeat a test multiple times"
+  [project n & tests]
```
Comment (jaor) [inline src/leiningen/test_repeat.clj]: it'd be nice to have a "repeat until it fails", with `n` being an upper bound to the number of executions... is that something doable easily in this same pr?

File: src/leiningen/test_children.clj [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        rows (mapv #(str %1 "\t" %2) (range) nses)
+        prompt "\nEnter namespaces to test (ex. 1 2 3 9 10 or 1-3 9 10): "
+        table (string/join "\n" (conj rows prompt))
+        resp (do (print table) (flush) (read-line))
+        indices (filterv #(< % (count nses)) (parse-selection resp))]
+    (map #(str (nses %)) indices)))
+
+(defn test-children
+  "Given a namespace prefix, either runs all unit tests in namespaces
+  matching t he prefix, or interactively select which matching
```
Comment (jaor) [inline src/leiningen/test_children.clj]: typo: the

File: README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Put `[test-helpers "0.1.0"]` into the `:plugins` vector of your `:user`
+profile.
+
+To install on a project-level:
+
+Put `[test-helpers "0.1.0"]` into the `:plugins` vector of your project.clj.
+
+Provides two lein tasks:
+
+### test-repeat
```
Comment (jaor) [inline README.md]: i'm only half-joking, but i wouldn't have resisted the temptation of naming this task iteratest :)

File: src/leiningen/test_children.clj [hunk 4]
```diff
@@ -0,0 +1,56 @@
+;; Copyright (c) 2016 BigML, Inc
+;; All rights reserved.
```
Comment (jaor) [inline src/leiningen/test_children.clj]: are you thinking of making this available in clojars? if so, you'll want to harmonize licenses...

**Comment by cheesinglee** [comment]
Added Apache license, changed behavior of test-repeat and gave it a non-boring name.

**Comment by jaor** [comment]
excellent. merged! i've also tagged and pushed it internally, but if you
want to put it up there in clojars, i'm all for it.

**Comment by jaor** [comment]
er, i meant i havent' yet pushed it internally: to do that, we'd better make this bigml/test-helpers so that we can use the bigml group.

**Comment by cheesinglee** [comment]
That's just a matter of changing the `defproject` right?

**Comment by jaor** [comment]
yes, i think so... and then i guess when one requires the plugin one has
to write [bigml/test-helpers "0.1.0"]...

**Comment by cheesinglee** [comment]
is that a change that requires review, or should I go ahead and just push it?

**Comment by jaor** [comment]
go ahead!

**Pull request merged**